Stellar-ledger.x

```diff
@@ -168,6 +175,16 @@ enum BucketEntryType
     INITENTRY = 2 // At-and-after protocol 11: only created.
 };
 
+enum HotArchiveBucketEntryType
+{
+    HA_METAENTRY = -1, // Bucket metadata, should come first.
```

Comment:
nit: shall we call this `HOST_ARCHIVE_...`? I think that's both more readable and safer (given that IIRC XDR enums are compiled into C enums).